site/specification/ORCv1.md

```diff
@@ -123,6 +123,16 @@ message Footer {
  repeated ColumnStatistics statistics = 7;
  // the maximum number of rows in each index entry
  optional uint32 rowIndexStride = 8;
+ // Each implementation that writes ORC files should register for a code
+ // 0 = ORC Java
+ // 1 = ORC C++
+ // 2 = Presto
+ // 3 = Scritchley Go from https://github.com/scritchley/orc
+ optional uint32 writer = 9;
+ // information about the encryption in this file
+ optional Encryption encryption = 10;
+ // the number of bytes in the encrypted stripe statistics
+ optional uint64 stripeStatisticsLength = 11;
```

Comment:
rename to encryptedStripeStatisticsLength ?

Reply:
My thought was that in ORCv2 we should move all of the stripe statistics into that section, so that all of the stripe statistics are split up by column. So that is why I didn't include the encrypted in the name. What do you think in that context?